Issue calling `go get` on repo.

Hey all,

I'm having an issue trying to `go get` this repo. Here's what I'm seeing.

```
adam@Planet-X -- burrow: (master) gpm install
>> Getting package github.com/samuel/go-zookeeper/zk
>> Getting package github.com/Shopify/sarama
>> Getting package github.com/cihub/seelog
>> Getting package code.google.com/p/gcfg
>> Getting package code.google.com/p/go-uuid
package code.google.com/p/go-uuid
	imports code.google.com/p/go-uuid
	imports code.google.com/p/go-uuid: no buildable Go source files in /Users/adam/src/go/src/code.google.com/p/go-uuid
>> Setting code.google.com/p/go-uuid to version 35bc42037350
>> Setting github.com/cihub/seelog to version 92dc4b8b540607b8187cc2f95cac200211dcd745
>> Setting code.google.com/p/gcfg to version c2d3050044d05357eaf6c3547249ba57c5e235cb
>> Setting github.com/Shopify/sarama to version e5e8ace555208299fd79a051b0910e56ce8d4cb9
>> Setting github.com/samuel/go-zookeeper/zk to version ad552be7b78b762b4a8040ffc5518bdaf5b7225d
>> Building package github.com/samuel/go-zookeeper/zk
>> Building package github.com/Shopify/sarama
../../Shopify/sarama/snappy.go:5:2: cannot find package "code.google.com/p/snappy-go/snappy" in any of:
	/usr/local/Cellar/go/1.4.2/libexec/src/code.google.com/p/snappy-go/snappy (from $GOROOT)
	/Users/adam/src/go/src/code.google.com/p/snappy-go/snappy (from $GOPATH)
```

Following that link leads me to (https://github.com/golang/snappy) which I'm able to call `go get github.com/google/go-snappy/snappy` fine, but this project seems to point back at the code.google.com version. So it seems that an upstream of a dependency here uses that?

--- a/Godeps
+++ b/Godeps
@@ -2,4 +2,4 @@ github.com/samuel/go-zookeeper/zk   ad552be7b78b762b4a8040ffc5518bdaf5b7225d
 github.com/Shopify/sarama           2ca3f4f9705b8391a9a1fe3e6ead0d57313108d9
 github.com/cihub/seelog             92dc4b8b540607b8187cc2f95cac200211dcd745
 code.google.com/p/gcfg              c2d3050044d05357eaf6c3547249ba57c5e235cb
-code.google.com/p/go-uuid           35bc42037350
+github.com/pborman/uuid             ca53cad383cad2479bbba7f7a1a05797ec1386e4

--- a/http_notifier.go
+++ b/http_notifier.go
@@ -12,7 +12,7 @@ package main
 
 import (
 	"bytes"
-	"code.google.com/p/go-uuid/uuid"
+	"github.com/pborman/uuid"
 	"encoding/json"
 	log "github.com/cihub/seelog"
 	"net/http"

